po/ja.po

```diff
@@ -1373,36 +1377,43 @@ msgid "**In-Person Event Checklist (Carpentries Workshops, CarpentryCon, Carpent
 "* Listen to the reporter and write down as many details as possible.\n"
 "* Ask the reporter to report the incident via the [Code of Conduct Incident Report Form][reporting-form]. If they would rather you complete the report, complete the report with their assistance. \n"
 "* Inform the workshop host that there was an incident and that a report was submitted via the [incident report form][reporting-form]. If the incident involves the workshop host, report the incident directly to the Code of Conduct committee via the incident report form. "
-msgstr ""
+msgstr "**対面イベントチェックリスト (カーペントリーズのワークショップ, CarpentryCon, CarpentryConnect, カーペントリーズのインストラクターのトレーニング)**\n"
+"* 参加者の安全を確保します。\n"
+"* そうでない場合は、即時対応チェックリストを参照してください。\n"
```

Comment:
This is missing the link to `#checklists-for-responding-to-an-incident`

Reply:
(Since we replaced the header from English to Japanese,) I think that this header link name would not link correctly. (so I removed that information.) I think it is better to check the new link URL (after the build) and add the link here.

Reply:
Added the link with 
https://github.com/kozo2/carpentries-coc/commit/ec39dd81bc75d5faf2f352e6b29ff14bc4918896